### ISSUE TITLE
Beta release

### ISSUE THREAD

**Issue by deepu105** [jhipster/jhipster-vuejs#158]
@jhipster/vuejs-developers do you think we are ready for a beta release for Christmas or New year? if so let me know and I'll prepare and do the release

**Comment by atomfrede** [comment]
I would go for new year so we can get the bundle size thing done.

**Comment by atomfrede** [comment]
So I would say when we have fixed #171 #164 and #72 we could do a first beta release, what do you think @jhipster/vuejs-developers ?

**Comment by sahbi-ktifa** [comment]
I agree, we have to focus on polishing then.

**Comment by pascalgrimaud** [comment]
Sorry @atomfrede 
I'd like a beta release soon, but I still saw some major issue right now.
I'll open new issues soon.

**Comment by atomfrede** [comment]
Yes I tried Matt's 21points entities they don't compile so at least that should work out of box even for a beta.

**Comment by pascalgrimaud** [comment]
did you add the blueprint vuejs flag ?

**Comment by atomfrede** [comment]
Yes it was a problem in the generated code with the user entity. Will post it tommorow when I am back a desktop/laptop.

**Comment by pascalgrimaud** [comment]
@atomfrede : just tested the 21points entities, and it works well for me. Excepting error message for validation min / max which is not always displayed

**Comment by atomfrede** [comment]
Mh strange, maybe I used an outdated version by accident, but good to know it works

**Comment by deepu105** [comment]
Ok guys, just let me know when you think its ready for beta. Keep in mind,
its beta so minor issues are ok

**Comment by deepu105** [comment]
Guys can we do a release today or tomorrow, it will nice for marketing

**Comment by sahbi-ktifa** [comment]
I'm working on prettier/lint today, I hope to finish it this morning. Either way, I think this would be a great news to do the release on this last day of the year :)

**Comment by pascalgrimaud** [comment]
We need to check if vuejs need last release or current master of generator-jhipster

Then, we need to fix this ticket https://github.com/jhipster/jhipster-vuejs/issues/76

**Comment by atomfrede** [comment]
It definitely needs the current master.

**Comment by sahbi-ktifa** [comment]
#190 is ready to be merged. Once master is released, we could do the beta version (after #76 indeed but I'm not sure about what to do first here)

**Comment by atomfrede** [comment]
Should we updatet to latest vue (and vue i18n) maybe before a beta release?

**Comment by sahbi-ktifa** [comment]
I'm afraid that the latest vue-template-compiler (2.5.21) version does not work well with typescript because it breaks out builds. We should stick to this one until at least 2.5.22 is out.

**Comment by sahbi-ktifa** [comment]
Without metrics support then?

**Comment by pascalgrimaud** [comment]
I don't think it is a must have, but it's my opinion

**Comment by sahbi-ktifa** [comment]
Ok, it can wait indeed 😁

**Comment by sahbi-ktifa** [comment]
@deepu105 @jdubois Do you plan to do a main generator release soon ? We could do a beta release then of the Vue.js blueprint.

**Comment by jdubois** [comment]
It's going to be complicated to do one this week. I need a couple of hours to do one, and I'm fully booked at the moment.

**Comment by yangliuyu** [comment]
Waiting for the beta release 😄

**Comment by jiangzhixiao** [comment]
Waiting for the first beta release too!

**Comment by riy** [comment]
Same here, I've postponed the implementation of a new app solely for the Vue.js blueprint :-)

**Comment by pascalgrimaud** [comment]
@atomfrede @sahbi-ktifa : we need to check if it works fine with jhipster 5.8.0, then I think we could do a beta release :)

**Comment by sahbi-ktifa** [comment]
Totally agreed! 😁

**Comment by deepu105** [comment]
Let me know when ready :)

**Comment by pascalgrimaud** [comment]
I think https://github.com/jhipster/jhipster-vuejs/issues/238 can wait. 

@sahbi-ktifa : what do you think about a release ? It will be in beta state so a release will give us more feedback.

**Comment by sahbi-ktifa** [comment]
@pascalgrimaud @deepu105 I think we can do this release now. It should give us more feedbacks even though we still have some work to do! But that's what beta are for 😉

**Comment by deepu105** [comment]
Ya sure, I'll take care of it. Ill do the release today

**Comment by deepu105** [comment]
Fred, since you own the npm repo you can also do the release if you like.
Just let me know if you prefer to do it, else ill do it

**Comment by atomfrede** [comment]
@deepu105 If you have time today I would suggest you do it as I don't know when I'll be home at my machine

**Comment by deepu105** [comment]
ok

Thanks & Regards,
Deepu

**Comment by deepu105** [comment]
Its done https://twitter.com/deepu105/status/1089959287093411840?s=20

Thanks & Regards,
Deepu
